packages/chaire-lib-backend/src/models/db/__tests__/users.db.test.ts

```diff
@@ -187,20 +212,25 @@ describe(`list users`, () => {
     });
 
     test('Get users list with email filter', async () => {
-        // Use 'test' string, should return 2 users
+        // Use 'test' string, should return 1 user as one email is modified before
         const { users: testUsers, totalCount: totalCountTest } = await dbQueries.getList({ filters: { email: 'test' }, pageIndex: 0, pageSize: -1 });
-        expect(totalCountTest).toEqual(2);
-        expect(testUsers.length).toEqual(2);
+        expect(totalCountTest).toEqual(1);
+        expect(testUsers.length).toEqual(1);
 
-        // Use 'test.org' string, should return 1 user
-        const { users: testOrgUsers, totalCount: totalCountTestOrg } = await dbQueries.getList({ filters: { email: 'test.org' }, pageIndex: 0, pageSize: -1 });
+        // Use 'example.com' string, should return 1 user
+        const { users: testOrgUsers, totalCount: totalCountTestOrg } = await dbQueries.getList({ filters: { email: 'example.org' }, pageIndex: 0, pageSize: -1 });
         expect(totalCountTestOrg).toEqual(1);
         expect(testOrgUsers.length).toEqual(1);
 
         // Use 'foo' string, should return nothing
         const { users: fooUsers, totalCount: totalCountFoo } = await dbQueries.getList({ filters: { email: 'foo' }, pageIndex: 0, pageSize: -1 });
         expect(totalCountFoo).toEqual(0);
         expect(fooUsers.length).toEqual(0);
+
+        // Use 'TEST' string, should return 1 user as it is case insensitive
+        const { users: capsUsers, totalCount: totalCountCaps } = await dbQueries.getList({ filters: { email: 'TEST' }, pageIndex: 0, pageSize: -1 });
+        expect(totalCountCaps).toEqual(1);
+        expect(capsUsers.length).toEqual(1);
```

Comment:
Here, you could add `expect(capsUsers).toEqual(testUsers)` as this is expected to be the same user in both queries. And for test flow, the caps case should be just below the initial `test` test.